lib/api/v3/users/users_api.rb

```diff
@@ -96,11 +94,13 @@ def user_transition(allowed)
             end
 
             patch do
+              authorize_logged_in
               authorize_admin
               update_user(request_body, current_user)
             end
 
             delete do
+              authorize_logged_in
               if ::Users::DeleteService.new(@user, current_user).call
```

Comment:
Again, checking authorization is already done in one row below in the `DeleteService`